front/src/services/filterService.ts

```diff
@@ -0,0 +1,41 @@
+export enum FilterType {
```

Comment:
data classes should be defined in schema package

Reply:
what is that?

Reply:
![image](https://user-images.githubusercontent.com/32846897/201885534-8eece1a7-cfb4-4dfd-8f84-6a07e86ded55.png)
Like this

Reply:
done